lm/vocabulary.py

```diff
@@ -185,12 +185,14 @@ def run(self):
                     words = line.strip().split()
                     counter.update(words)
 
-        cutoff = min(self.num_words, len(counter))
+        num_words = self.num_words if isinstance(self.num_words, int) else self.num_words.get()
```

Comment:
I would turn this check around to be more permissive for ducktyping
```suggestion
        num_words = self.num_words.get() if isinstance(self.num_words, tk.Variable) else self.num_words
```